Tests/TestingApps/PaywallsTester/PaywallsTester/Extensions/UserDefaultsExtensions.swift

```diff
@@ -0,0 +1,18 @@
+//
+//  UserDefaults.swift
+//
+//
+//  Created by Nacho Soto on 12/11/23.
+//
+
+import Foundation
+
+public extension UserDefaults {
+
+    static let revenueCatSuite: UserDefaults = .init(
+        suiteName: sharedAppGroup
+    )!
+
+    static let sharedAppGroup: String = "group.com.revenuecat"
```

Comment:
do we need the app group for anything?

Reply:
I kept it because in the future if we made any other apps it would allow us to share data, including logged-in status across all of our apps. This for example is how Google's apps work, where if you're logged in on one and download another you can sign in without any sort of challenge. Thoughts?

Reply:
can we migrate into it later? or is it a pain if we don't do it now?

Reply:
I don't think it'd be a problem to do it later other than old versions of the app would be left out of the group. But it's not hard to do it now either, it's doing it now.

Reply:
sounds good, let's keep it then